src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -998,7 +999,9 @@ struct FixedKVLeafNode
   // 2. The child extent is stable
   bool is_child_stable(uint16_t pos) const final {
     auto child = this->children[pos];
-    if (is_valid_child_ptr(child)) {
+    if (is_reserved_ptr(child)) {
+      return true;
+    } else if (is_valid_child_ptr(child)) {
       ceph_assert(child->is_logical());
```

Comment:
It crashed at line 1008: https://github.com/ceph/ceph/blob/34f4f60094eaab6f8512135dac89a9e56f59a1e3/src/crimson/os/seastore/btree/fixed_kv_node.h#L1008

It was because that the reserved mapping was actually a newly inserted mapping by the transaction that caused the crash and fixed-kv-node didn't check if it's a reserved one.